airbyte-workers/src/main/java/io/airbyte/workers/process/KubeProcessFactory.java

```diff
@@ -91,9 +79,11 @@ public KubeProcessFactory(final WorkerConfigs workerConfigs,
   }
 
   @Override
-  public Process create(final String jobId,
+  public Process create(
+                        final String jobType,
+                        final String jobId,
                         final int attempt,
-                        final Path jobRoot, // todo: remove unused
```

Comment:
I removed this comment since we cannot remove this as it's used by the docker implementation of this interface.